Shared/qml/AlertConditionsTool.qml

```diff
@@ -244,6 +248,8 @@ DsaPanel {
             width: parent.width
             spacing: 10 * scaleFactor
             leftPadding: 10 * scaleFactor
+            property bool nameAlreadyInUseMobile: !toolController.conditionAlreadyAdded(editConditionMobileName.text)
+            property bool validMobile: editConditionMobileName.length > 0 && nameAlreadyInUseMobile
```

Comment:
This is confusing to me. It is valid if it has some length and is already in use?

Is `nameAlreadyInUseMobile: !toolController.conditionAlreadyAdded...` correct?

Reply:
Yep, this logic was named wrong/backwards. I started it on the main naming page for the alert and somehow managed to transpose the logic here. Was made to work properly, just used backwards for its name.

Reply:
added commit to fix logic
9dbf34e0e5ddb97538963ac073c2bd74dc0f7741